reactive/etcd.py

```diff
@@ -51,6 +53,14 @@ def check_cluster_health():
     status_set('active', health['status'])
 
 
+@when('etcd.installed')
+def set_app_version():
+    # sneak in the version as well
+    cmd = ['etcd', '-version']
+    version = check_output(cmd).split(b'\n')[0].split(b':')[-1]
```

Comment:
This change looks fine to me, but I would recommend putting the version format that is __currently__ returned in the comment above the split to show the developer what you expected and why it works. That way if the version ever changed in a way that breaks this logic we can begin to see what you had expected and what it returns now.